### PR TITLE
Do you speak PS? April 17 edition

### PR DESCRIPTION
**Pull request by AlexEven**

### CODE REVIEW

File: news/_posts/translation/2017-05-19-do-you-speak-prestashop-april-2017.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+Contributing to PrestaShop is not only about the code, it's also taking part to the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! Read this report to see how the translations of the software evolved in April.
+
+![Crowdin Monthly banner](/assets/images/2017/04/DYSpeakPS.jpg)
+
+## Project news
+
+### Spanish tone harmonization
+
+A good news is that the Spanish translation is now 100% complete! However, there is sometimes a mix of polite versus familiar tone (tu/usted) that needs harmonization.
+To be aligned with the global tone of PrestaShop, we're chosing to use 'tu' when talking the to merchants in Spanish (= in the back office). From now on, all the translations from the back office ('Admin' domain) must use an informal tone (tu).
```
Comment (xBorderie) [inline news/_posts/translation/2017-05-19-do-you-speak-prestashop-april-2017.md]: "choosing"
"the to"

File: news/_posts/translation/2017-05-19-do-you-speak-prestashop-april-2017.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+This doesn't impact the front office ('Shop' domains) that remains polite by default. We leave it to the merchants to change the tone of their shop, should they wish it.
+
+As a consequence:
+
+* We'll modify existing back office translation to harmonize the tone right away
```
Comment (xBorderie) [inline news/_posts/translation/2017-05-19-do-you-speak-prestashop-april-2017.md]: "translations".

File: news/_posts/translation/2017-05-19-do-you-speak-prestashop-april-2017.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+* New back office translations must use the informal tone from now on.
+
+
+### Contributor night in Paris
+
+For those of you in Paris on the 14th of June, the night before the [PrestaShop Day Paris](http://www.prestashopday.com/fr/) will be dedicated to community contributions. Come and meet the PrestaShop team to take part in the PrestaShop project (code, documentation or translations). [More information here](http://build.prestashop.com/news/contributor-night-barcamp-prestashop-day-paris-2017/)!
```
Comment (xBorderie) [inline news/_posts/translation/2017-05-19-do-you-speak-prestashop-april-2017.md:32]: Merci ! :)

File: news/_posts/translation/2017-05-19-do-you-speak-prestashop-april-2017.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+| Russian | Slovak | Slovene | Spanish |
+| Swedish | Turkish | Ukrainian |
+
+That's 2 extra languages: Croatian and Spanish, thank you!
+And version 1.7 is well translated in 11 more languages. Soon at 100% we hope :)
+
+
+## Languages with the best evolution
+
+In comparison with March 2017, the following languages had the best progress thanks to the translation community:
```
Comment (xBorderie) [inline news/_posts/translation/2017-05-19-do-you-speak-prestashop-april-2017.md:112]: April 2017?

Comment (AlexEven) [reply news/_posts/translation/2017-05-19-do-you-speak-prestashop-april-2017.md:112]: April report, published in May ;)

**Pull request merged**